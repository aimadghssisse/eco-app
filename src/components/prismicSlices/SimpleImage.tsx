/**
 * Prismic Slice - Landing page Hero Component
 */

import * as React from "react"

const SimpleImage = ({ data }) => {
    return (
        <div className="text-center mt-6 flex justify-center">
            <img src={data.primary.image.url} alt={data.primary.image.alt} />
        </div>
    )
}

export default SimpleImage
