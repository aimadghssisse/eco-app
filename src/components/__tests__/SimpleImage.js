import React from "react"
import renderer from "react-test-renderer"

import SimpleImage from "../prismicSlices/SimpleImage"

describe("SimpleImage", () => {
    it("renders correctly", () => {
        const tree = renderer
            .create(
                <div className="text-center">
                    <img className="mt-6" src="data.primary.image.url" alt="data.primary.image.alt" />
                </div>
            )
            .toJSON()
        expect(tree).toMatchSnapshot()
    })
})