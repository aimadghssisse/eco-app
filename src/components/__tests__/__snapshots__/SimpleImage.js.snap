// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SimpleImage renders correctly 1`] = `
<div
  className="text-center"
>
  <img
    alt="data.primary.image.alt"
    className="mt-6"
    src="data.primary.image.url"
  />
</div>
`;

exports[`SimpleImage renders correctly 1`] = `
<div
  className="text-center"
>
  <img
    alt="data.primary.image.alt"
    className="mt-6"
    src="data.primary.image.url"
  />
</div>
`;
